sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_datastore/credentials.py

```diff
@@ -53,7 +53,7 @@ def predump(self, data, **kwargs):
 
 
 class BaseTenantCredentialSchema(metaclass=PatchedSchemaMeta):
-    authority_url = fields.Str(data_key="authority_uri")
+    authority_url = fields.Str()
```

Comment:
Will this break customers' existing YAML files?

Reply:
I haven't seen anyone use `authority_uri` or purposefully set it to something but it makes sense to not break existing work flows for customers.

Added backward compatibility to also support `authority_uri` and tests for validation.